akka-persistence-typed/src/main/scala/akka/persistence/typed/delivery/EventSourcedProducerQueue.scala

```diff
@@ -268,7 +272,9 @@ private class EventSourcedProducerQueue[A](
                 _.confirmationQualifier != confirmationQualifier) =>
             confirmationQualifier
         }.toSet
-        if (old.isEmpty) {
+        val stateWithoutPartialChunkedMessages = state.cleanupPartialChunkedMessages()
```

Comment:
Great catch. It's actually more difficult than just looking at the flag, since it's possible that another command arrives before the initial CleanupTick. Solved in 381f2bf